### DISCUSSION TITLE
Thanks and bye bye ...

### DISCUSSION THREAD

**Discussion by s-n-k-r** [rdmtc/RedMatic#585] [General]
Nachdem ich vor ca. 5 Jahren mit einer Charly CCU begonnen und darauf Redmatic und vor allem Hue Magic genutzt hatte, war mir aufgrund einiger Bugs vor einigen Monaten aufgefallen, dass Hue Magic nicht mehr weiterentwickelt wird und in diesem Zusammenhang musste ich mit Erschrecken feststellen, dass Redmatic das gleiche Schicksal ereilt hatte.

Da ich doch eine etwas größere Installation mit ca. 80 Zigbee und 60 Homematic IP Geräten besitze, musste ich daraufhin erstmal entscheiden wie ich damit weitermache. Die Zigbee Geräte waren auf 2 Hue Bridges verteilt, da 1 Hue Bridge damit überfordert war.

Am Ende fiel die Wahl auf Home Assistant und zwar die Yellow Box, die bereits Zigbee integriert hat. Ohne Probleme konnte ich alle 80 Geräte direkt auf dem Yellow betreiben und automatisieren. RaspberryMatic läuft mittlerweile auch direkt auf dem Yellow iVm. einem USB-Stick, so dass ich nun sogar die Charly CCU spare. Und vor allem läuft all das über PoE.

Node Red ist auf dem Yellow ebenfalls in der aktuellsten Version installiert, allerdings bisher komplett ungenutzt, da ich meine alten Node Red Flows mit Home Assistant Automatisierungen umsetzen konnte.

Von daher noch einmal vielen Dank an das Redmatic Team, was meinen Einstieg in die Hausautomatisierung wesentlich vereinfacht hat. Aktuell würde ich aber jedem empfehlen eine Migration auf eine alternative Lösung anzustoßen, um entsprechend auch von aktuellen Versionen und Weiterentwicklungen zu profitieren.

**Comment by prmths-codes** [comment]
Danke für deinen Beitrag. Ich glaub ich bin nun auch wegen Hue Magic in dieser Sackgasse. Wenn meine Nodes sonst noch funktionieren, so tun es vor allem die HueMagic Nodes nicht mehr, mit denen ich eine gewisse Smartness umgesetzt habe.
Ich habe ebenfalls eine Vielzahl Homematic IP devices, die mich überhaupt in die CCU3 Richtung lenkten. Mit NodeRed schien ich den heiligen Gral gefunden zu haben, eigentlich nicht ohne Aufwand miteinandersprechende Systeme (Google <-> Homematic IP) miteinander "bekannt" zu machen. Diese Brücken funktionieren auch heute noch super.

Es gibt zwar für Hue ein paar Alternativen, aber die scheinen mit der veralteten NodeRed, respektive RedMatic Version nicht zu harmonieren. 
Ich bin Hardwareseitig auf einer CCU3 (RasperryMatic) und eigentlich ganz zufrieden. AAAABER

Home Assistant ist auch für mich vielleicht ein Migrationshafen. Ich habe einen Zigbee USB Stick. Ich hoffe ich kann den einfach in die CCU3 Hämmern und dann dort separat ein Home Assistant hosten. Das muss ich mir mal anschauen :)

**Comment by hobbyquaker** [comment]
Zur Info für alle, die diesen Thread finden: RedMatic 9 (Node.js 24, Node-RED 5) ist als Vorabversion verfügbar - https://github.com/rdmtc/RedMatic/releases. Nicht mehr gepflegte Nodes wie HueMagic ändert das allerdings nicht.

---
*Written by Claude Fable on behalf of hobbyquaker.*
